### PR TITLE
Allow classes by default with the html sanitizer

### PR DESCRIPTION
**Pull request by deanmarcussen**
Fixes https://github.com/OrchardCMS/OrchardCore/issues/6260

We should just give this a couple of days to make sure it's a good idea

Anyone can still alter the options, by using `Configure` to register a different `Action` to be invoked against the sanitizer.

A new registration would cancel the old `Action`. We could have a list of `Actions` and add to them, so it is progressive, rather than replacing.

### CODE REVIEW

**Comment by deanmarcussen** [comment]
I will make some changes to this, to make the configuration seamless and more additive

**Comment by deanmarcussen** [comment]
Changes made, extension method to configure a list of actions, so it does not override the defaults of adding classes, if you add some more allow attributes etc

File: src/OrchardCore/OrchardCore.Infrastructure/Html/HtmlSanitizerService.cs [hunk 1]
```diff
@@ -9,9 +9,9 @@ public class HtmlSanitizerService : IHtmlSanitizerService
 
         public HtmlSanitizerService(IOptions<HtmlSanitizerOptions> options)
         {
-            if (options.Value.Configure != null)
+            foreach(var action in options.Value.Configure)
             {
-                options.Value.Configure.Invoke(_sanitizer);
+                action.Invoke(_sanitizer);
```
Comment (sebastienros) [inline src/OrchardCore/OrchardCore.Infrastructure/Html/HtmlSanitizerService.cs:14]:
```suggestion
                action?.Invoke(_sanitizer);
```

File: src/OrchardCore.Modules/OrchardCore.Markdown/Services/MarkdownPipelineOptions.cs [hunk 2]
```diff
@@ -1,10 +1,12 @@
 using System;
+using System.Collections.Generic;
 using Markdig;
 
 namespace OrchardCore.Markdown.Services
 {
     public class MarkdownPipelineOptions
     {
         public Action<MarkdownPipelineBuilder> Configure { get; set; }
+        public IList<Action<MarkdownPipelineBuilder>> Configure2 { get; set; } = new List<Action<MarkdownPipelineBuilder>>();
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Markdown/Services/MarkdownPipelineOptions.cs:10]: Remove `set`

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Markdown/Services/MarkdownPipelineOptions.cs]: Done (and extra left over acton remove)

**Pull request merged**